lib/donations/currencies.rb

```diff
@@ -0,0 +1,47 @@
+# == Currency Converter to JSON
+#
+# Takes an array of integer amounts, in US cents, and
+# converts to GBP and EUR.
+#
+# ==== Examples
+#
+#   currencies = Donations::Currencies.for(102, 204, 302, 401)
+#   currencies.to_hash # also to_json
+#   =>
+#   {
+#     USD: [1.02, 2.04, 3.02, 4.01],
```

Comment:
@NealJMD that sounds good for now. I think this is something we'll want to test when up and running, but like the idea of defaulting to nearest 5 when above 20 unites.